pyaedt/modules/solutions.py

```diff
@@ -1071,338 +1075,248 @@ def ifft_to_file(
 
 
 class FfdSolutionData(object):
-    """Contains Hfss Far Field Solution Data (ffd)."""
+    """Contains information from the far field solution data.
+    Load far field data from the element pattern files.
+
+    Parameters
+    ----------
+    eep_files : list or str
+        List of element pattern files for each frequency.
+    frequencies : list, str, int or float
```

Comment:
```suggestion
    frequencies : list, str, int, or float
```
We use Oxford commas, which are placed before the "and" or the "or" in a list of items.